dsymbol/src/dsymbol/conversion/first.d

```diff
@@ -37,6 +37,7 @@ import std.experimental.allocator;
 import std.experimental.allocator.gc_allocator : GCAllocator;
 import std.experimental.logger;
 import std.typecons : Rebindable;
+import std.array : appender;
```

Comment:
unused import

Reply:
```
src/dsymbol/conversion/first.d(788,14): Error: template instance `appender!string` template `appender` is not defined
src/dsymbol/conversion/first.d(1072,14): Error: template instance `appender!string` template `appender` is not defined
src/dsymbol/conversion/first.d(1142,16): Error: template instance `appender!string` template `appender` is not defined
src/dsymbol/conversion/first.d(1346,13): Error: template instance `appender!string` template `appender` is not defined
```

its used here

Reply:
oh right, it's the appender moved up from the child methods